src/traces/heatmap/attributes.js

```diff
@@ -134,8 +134,6 @@ module.exports = extendFlat({
 
     showlegend: extendFlat({}, baseAttrs.showlegend, {dflt: false}),
     zorder: scatterAttrs.zorder
-}, {
-    transforms: undefined
-},
+}, {},
```

Comment:
```suggestion
},
```
This is an `extendFlat` call, so an empty object is a noop.